src/silo/query_engine/actions/aggregated.cpp

```diff
@@ -0,0 +1,300 @@
+#include "silo/query_engine/actions/aggregated.h"
+
+#include <algorithm>
+#include <chrono>
+#include <functional>
+#include <random>
+#include <utility>
+#include <vector>
+
+#include <tbb/enumerable_thread_specific.h>
+#include <tbb/parallel_for_each.h>
+#include <boost/algorithm/string/join.hpp>
+#include <nlohmann/json.hpp>
+
+#include "silo/common/hashing.h"
+#include "silo/database.h"
+#include "silo/query_engine/operator_result.h"
+#include "silo/query_engine/query_parse_exception.h"
+#include "silo/query_engine/query_result.h"
+#include "silo/storage/metadata_store.h"
+
+using silo::common::Hasher;
+
+namespace silo::query_engine::actions {
+
+struct Tuple {
+   std::vector<std::string> string_columns;
+   std::vector<uint32_t> int_columns;
```

Comment:
Are these the values in the int_columns/string_columns? Should we name it so it repersents that?